packages/mui-codemod/src/deprecations/backdrop-props/test-cases/actual.js

```diff
@@ -8,17 +8,18 @@ import { Backdrop as MyBackdrop } from '@mui/material';
   slots={{
     root: 'div',
   }}
-  slotProps={{
```

Comment:
`slotProps` is not modified in the `backdrop-props` codemod, so there's no need for it on the test case. Same for line 20.